storybook/storybook-react/src/Grid/Grid.docs.mdx

```diff
@@ -29,3 +43,7 @@ Elke cel biedt ruimte voor een afbeelding.
 <Canvas of={GridStories.Cells} />
 
 Voor meer voorbeelden raadpleeg je [Grid Cell](?path=/docs/react_layout-grid-cell--docs) zelf.
+
+## Richtlijnen
+
+- Gebruik de `marginVertical` prop niet tegelijk met de `marginTop` of `marginBottom` props.
```

Comment:
Is this necessary to document? Typescript already prevents you from doing this, no?

Reply:
But this is React specific documentation? Feels like it's not necessary to add it here. Of course, it doesn't really hurt either, so we can keep it if you feel it adds something.

Reply:
Allright, removed both.